Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -1214,6 +1214,7 @@ def prepGlyph(g, marker, index=0):
     if t == 'dot':
         gs.SetGlyphTypeToCircle()
         gs.FilledOn()
+        s *= numpy.pi
```

Comment:
Why not make it a integer number say 10, 20? it may help if we are going to do pixel align rendering.